src/software/ai/hl/stp/play/shoot_or_pass/shoot_or_pass_play_fsm.cpp

```diff
@@ -84,7 +82,7 @@ void ShootOrPassPlayFSM::lookForPass(const Update& event)
                              pass_score_ramp_down_duration,
                          1.0 - abs_min_pass_score);
         updateOffensivePositioningTactics(ranked_zones, pass_eval,
-                                          event.common.num_tactics - 1);
+                                          event.common.num_tactics - 2);
```

Comment:
what does this change do / how does it work?

Reply:
so this makes it so that we have 1 attacker, 1 receiver, and num_tactics - 2 supporting robots

Reply:
Gotcha, can u add a comment for that?

Reply:
bump